setup/emscripten/index.html.mako

```diff
@@ -0,0 +1,55 @@
+## -*- coding: utf-8 -*-
+<%inherit file="/_templates/markdown.mako" />
+
+emscripten setup
+===========
+
+Welcome to processing.js for Open Frameworks! Have you ever made a project in openFrameworks and been like "This is awesome! I want to put it on my online portfolio!". You could always make a video or take photos, but what about the UI aspect of your project? How can people see the awesome mouse effects you made? Enter Emscripten. Emscripten turns your C++ code into Javascript code in a matter of minutes! So you can put your awesome OF project "on the line" in your portfolio.
```

Comment:
i would reword this, processing.js is written in javascript by the user while what we do is translate c++ to javascript so it might be confusing. also Open Frameworks -> openFrameworks